src/main/java/net/hypixel/nerdbot/generator/impl/tooltip/MinecraftTooltipGenerator.java

```diff
@@ -82,10 +82,12 @@ public MinecraftTooltip parseLore(String input, TooltipSettings settings) {
             .withAlpha(Range.between(0, 255).fit(settings.getAlpha()));
 
         if (settings.getName() != null && !settings.getName().isEmpty()) {
-            String name = settings.getName();
+            String name;
 
             if (rarity != null && rarity != Rarity.byName("NONE")) {
-                name = rarity.getColorCode() + name;
+                name = rarity.getColorCode() + settings.getName();
+            } else {
+                name = settings.getName();
```

Comment:
I still don't see the point in doing this because it doesn't really change anything